lib/resources/nginx_conf.rb

```diff
@@ -93,13 +96,37 @@ def resolve_references(data, rel_path)
       if data.key?('include')
         data.delete('include').flatten
             .map { |x| File.expand_path(x, rel_path) }
+            .map { |x| find_files(x) }.flatten
```

Comment:
Thank you for the fix Jerry. 
However this solution does not always include files (included with '*') because of issue#2135 (find_files uses the command resource).
I just tested this code over an ssh connection and it does not parse all the included conf files;
It works fine when I tested by changing the find_files code to use bash resource instead of command resource.
Hope you can find a solution for it.

Thank you again Jerry,
Rony

Reply:
I know we have a logged issue about the command resource specifically, but we should log an issue for the `find_files` method too so we don't lose that context, and link it to this PR. I'm not going to hold off merging this PR since it's still a step in the right direction and will work fine for users not executing via sudo.

Reply:
https://giphy.com/gifs/reaction-LSmULmByAQHQs/fullscreen

Reply:
Haha sorry @jerryaldrichiii . 
Thank you @adamleff I have posted and issue for find_files. issue #2157